### PR TITLE
InitCommand: ask for confirmation before overwriting potentially existing deploy.[php/yaml]

### PR DESCRIPTION
**Pull request by thyseus**
if deploy.[php/yaml] already exists

- [X] New feature?
- [X] Changelog updated?

### CODE REVIEW

File: src/Command/InitCommand.php [hunk 1]
```diff
@@ -100,7 +109,13 @@ protected function execute(InputInterface $input, OutputInterface $output): int
         if (isset($tempHostFile)) {
             $host = file_get_contents($tempHostFile);
         }
-        $hosts = explode(',', $io->ask('Hosts (comma separated)', $host));
+        $hosts = [];
+        $answer = $io->ask('Hosts (comma separated)');
```
Comment (antonmedv) [inline src/Command/InitCommand.php:113]: Why this change? Also looks like `$host` default value is missing.

**Review by antonmedv — CHANGES_REQUESTED** [review]

File: src/Command/InitCommand.php (hunk 1 above)
Comment (thyseus) [reply src/Command/InitCommand.php:113]: Behavior before this change:

```
 Hosts (comma separated):
 > 

 TypeError  in InitCommand.php on line 103:

  explode() expects parameter 2 to be string, null given
  ```
  
  Behavior after this change:
  
  ```
   Hosts (comma separated):
 > 

No hosts given, you need to define them manually in deploy.php.
Successfully created deploy.php
```

`$host` is initialized in line 108.

Comment (antonmedv) [reply src/Command/InitCommand.php:113]: $host is initialized but not used. It should go to `$io->ask('Hosts (comma separated)', $host)`.

Comment (thyseus) [reply src/Command/InitCommand.php]: Ah i see. It is the default value of the potential guess. I added it to the appropriate place.

**Comment by antonmedv** [comment]
Thanks! I added it manually.
